core/src/main/java/com/graphhopper/routing/util/DataFlagEncoder.java

```diff
@@ -156,6 +164,18 @@ public int defineWayBits(int index, int shift) {
         carBwdMaxspeedEncoder = new EncodedDoubleValue("car bwd maxspeed", shift, speedBits, speedFactor, 0, maxPossibleSpeed, true);
         shift += carBwdMaxspeedEncoder.getBits();
 
+        /* Value range: [3.0m, 5.4m] */
```

Comment:
I think I leave the current encoders for road attributes as-is, with only one minor improvement in the latest commit. If the value is too big for the encoder, we just log & ignore, and continue parsing the OSM data.

Given this change, I was able to parse Australian OSM with only few exceptions of "too large value".